torchao/prototype/float8nocompile/float8nocompile_linear.py

```diff
@@ -19,4 +36,149 @@ class Float8LinearNoCompile(torch.nn.Linear):
     Note: this is **prototype** and not suitable for production use.
     """
 
-    pass
+    def __init__(self, *args, **kwargs):
+        """
+        Additional arguments on top of `torch.nn.Linear`'s arguments:
+        * `config`: Float8LinearConfig
+        """
+
+        # Amax scales should always be kept as float32.
+        self.always_float32_buffers = set()
```

Comment:
can delete, buffers are only needed for delayed scaling

Reply:
ah yes I noticed buffers weren't needed for dynamic scaling but missed this one somehow - thanks!